apps/spruce/src/test_utils/utils.ts

```diff
@@ -33,4 +11,4 @@ const selectLGOption = async (dataCy: string, option: string) => {
   expect(screen.queryByDataCy(dataCy)).toHaveTextContent(option);
 };
 
-export { mockEnvironmentVariables, selectLGOption };
+export { selectLGOption };
```

Comment:
[request] Could we fix the errors in `selectLGOption`, it's really easy just replace 
`screen.queryByDataCy` => `screen.getByDataCy`
`screen.queryByText` => `screen.getByText`